react/components/FooterSocialNetworkList.js

```diff
@@ -1,5 +1,5 @@
 import PropTypes from 'prop-types'
-import React, { Component } from 'react'
+import React from 'react'
```

Comment:
It is better to use PureComponent in cases like this. See the [documentation](https://reactjs.org/docs/react-api.html#reactpurecomponent) for more details.